src/code_190B20.c

```diff
@@ -42,9 +46,13 @@ INCLUDE_ASM(s32, "code_190B20", func_80263E08);
 
 INCLUDE_ASM(s32, "code_190B20", set_animation_rate);
 
-INCLUDE_ASM(s32, "code_190B20", set_actor_yaw);
+void set_actor_yaw(s32 actorId, s32 yaw) {
+    get_actor(actorId)->yaw = yaw;
```

Comment:
we should really decide what we're doing with these weird accesses. I hate them, but if everyone else wants them, then we should go for it